### PR TITLE
Moves tab styling

### LINKED ISSUE #84: Moves Tab Styling
**Is your feature request related to a problem? Please describe.**
Moves tab in modal content needs to be displayed better.

**Describe the solution you'd like**
Format and style content to be more user friendly.

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Implemented overall styling to moves tab
2. Use Chart.js library to display moves stats
3. Function to check for null values and change to 0
4. Imported move type icons and assigned to moves according to type

## Purpose
For the content on the moves tab to be displayed in a more user friendly way.

## Approach
Implemented chart.js for data visuals

https://www.chartjs.org/

Closes #84

### CODE REVIEW

File: src/util/fetchPokemonData.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const Pokedex = require("pokeapi-js-wrapper");
 
 const interval = {
   offset: 0,
-  limit: 50,
+  limit: 5,
   // Remove line above and uncomment the line below after adding lazy loading
   // limit: 1119
```
Comment (BlackCubes) [inline src/util/fetchPokemonData.js:8]: I'm assuming you might need dis 👀?

**Review by BlackCubes — APPROVED** [review]
Looks good! Can't wait to see y'all use Chart.js!

File: src/util/fetchPokemonData.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const Pokedex = require("pokeapi-js-wrapper");
 
 const interval = {
   offset: 0,
-  limit: 50,
+  limit: 5,
   // Remove line above and uncomment the line below after adding lazy loading
```
Comment (krferrell) [inline src/util/fetchPokemonData.js:7]: Lazy loading? Nice! I have been looking into different ways of achieving this.

**Review by krferrell — APPROVED** [review]
Great job. Everything looking good. I like how you handled importing the images.

**Pull request merged**